### ISSUE TITLE
Is that normal that logits are not within -1 to 1?

### ISSUE THREAD

**Issue by Leo-T-Zang** [openai/CLIP#444]
Hi all,

I tried to train CLIP model on my dataset and reached over 0.8 top-1 accuracy. When I print all logits values out, I notice there are some values not within [-1,1] range, which is supposed to be the range of cosine similarity. 

Is this normal or potentially I did anything wrong?

Thanks a lot!
